Intersect (Core)/Configuration/ClientConfiguration.cs

```diff
@@ -69,58 +81,98 @@ public void Validate()
         #region Options
 
         /// <summary>
-        /// Hostname of the server to connect to
+        /// Hostname of the server that the client will connect to.
         /// </summary>
         public string Host { get; set; } = DEFAULT_HOST;
 
         /// <summary>
-        /// Port of the server to connect to
+        /// Port of the server that the client will connect to.
         /// </summary>
         public ushort Port { get; set; } = DEFAULT_PORT;
 
         /// <summary>
-        /// The font family to use on misc non-ui rendering
+        /// The font family to use on misc non-ui rendering.
         /// </summary>
         public string GameFont { get; set; } = DEFAULT_FONT;
 
         /// <summary>
-        /// The font family to use on entity names
+        /// The font family to use on entity names.
         /// </summary>
         public string EntityNameFont { get; set; } = DEFAULT_FONT;
 
         /// <summary>
-        /// The font family to use on chat bubbles
+        /// The font family to use on chat bubbles.
         /// </summary>
         public string ChatBubbleFont { get; set; } = DEFAULT_FONT;
 
         /// <summary>
-        /// The font family to use on action messages
+        /// The font family to use on action messages.
         /// </summary>
         public string ActionMsgFont { get; set; } = DEFAULT_FONT;
 
         /// <summary>
-        /// The font family to use on unstyled windows such as the debug menu/admin window
+        /// The font family to use on un-styled windows such as the debug menu/admin window.
         /// </summary>
         public string UIFont { get; set; } = DEFAULT_UI_FONT;
 
         /// <summary>
-        /// Number of lines to save for chat scrollback
+        /// Number of lines to save for chat scrollback.
         /// </summary>
         public int ChatLines { get; set; } = DEFAULT_CHAT_LINES;
 
         /// <summary>
-        /// Menu music file name
+        /// Sets a music file to be played in the main menu.
         /// </summary>
         public string MenuMusic { get; set; } = DEFAULT_MENU_MUSIC;
 
         /// <summary>
-        /// Menu background art
+        /// Sets a list from which introductory images will be drawn when the game client starts.
+        /// </summary>
+        public List<string> IntroImages { get; set; } = new List<string>();
```

Comment:
None of these documentation changes belong in this PR, they should be a separate PR.